sphinx/index.rst

```diff
@@ -64,3 +64,10 @@ Indices
 
 * :ref:`genindex`
 * :ref:`modindex`
+* :ref:`terms`
+
+.. toctree::
+   :hidden:
+   :name: hidden-index
+
+   terms
```

Comment:
Introduces new build `WARNING: undefined label: terms (if the link has no caption the label must precede a section header)`?